test/inductor/test_cpu_select_algorithm.py

```diff
@@ -431,6 +431,92 @@ def forward(self, x):
+    @torch.no_grad
+    @unittest.skipIf(not TEST_MKL, "Test requires MKL")
+    @parametrize("batch_size", (32,))
+    @parametrize("in_features", (128,))
+    @parametrize("out_features", (64,))
+    @parametrize("bias", (False, True))
+    @parametrize("input_3d", (False, True))
+    @parametrize("int8_mixed_bf16", (False, True))
+    @dtypes(torch.float32, torch.bfloat16)
+    @parametrize(
+        "epilogue",
+        (
+            "none",
+            "relu",
+        ),
+    )
+    def test_quantized_linear_with_pointwise_binary(
+        self,
+        batch_size,
+        in_features,
+        out_features,
+        bias,
+        input_3d,
+        int8_mixed_bf16,
+        dtype,
+        epilogue,
+    ):
+        if not int8_mixed_bf16 and dtype == torch.bfloat16:
+            return
+        B = (2, batch_size) if input_3d else (batch_size,)
+        input = torch.randn(*B, in_features).to(dtype=torch.float32)
+
+        other = torch.randn(*B, out_features).to(dtype=dtype)
+        other2 = torch.randn(*B, out_features).to(dtype=dtype)
+        if not input_3d:
+            # Avoid hiting qlinear inplace sum fusion
+            other2 = torch.randn(1, *B, out_features).to(dtype=dtype)
+
+        class M(torch.nn.Module):
+            def __init__(self, bias, input_3d):
+                super().__init__()
+                self.linear = torch.nn.Linear(in_features, out_features, bias)
+                self.epilogue = _get_epilogue(epilogue)
+                self.linear2 = torch.nn.Linear(out_features, out_features, bias)
+                self.epilogue2 = _get_epilogue(epilogue)
+                self.input_3d = input_3d
+
+            def forward(self, x, other, other2):
+                res = self.epilogue(self.linear(x) + other)
+                if not self.input_3d:
+                    # Avoid hiting qlinear inplace sum fusion
+                    other2 = other2.view(other2.size(1), other2.size(2))
+                res = self.epilogue2(self.linear2(res) + other2)
+                return res
+
+        counters.clear()
+        ref_quantized_mod = _generate_qdq_quantized_model(
+            M(bias=bias, input_3d=input_3d).eval(),
+            (input, other, other2),
+        )
+        atol, rtol = 5e-2, 5e-2
+        with patch.object(
+            select_algorithm, "VERIFY", dict(atol=atol, rtol=rtol)
+        ), torch.no_grad(), torch.autocast(
+            "cpu", enabled=int8_mixed_bf16, dtype=torch.bfloat16
+        ):
+            ref_res = ref_quantized_mod(input, other, other2)
+            cfn = torch.compile(ref_quantized_mod)
+            res = cfn(input, other, other2)
+            self.assertEqual(
+                res,
+                ref_res,
+                atol=atol,
+                rtol=rtol,
+                equal_nan=True,
+                exact_dtype=True,
+            )
+            self.assertEqual(
+                counters["inductor"]["select_algorithm_autotune"],
+                2,
+            )
```

Comment:
self.assertEqual(counters["inductor"]["cpp_epilogue_fusion_counter"], 1)

Reply:
Thanks for the comment. Added `self.assertEqual(counters["inductor"]["cpp_epilogue_fusion_counter"], 0)` since `add` and `add_relu` should be post op fusion as gemm template instead of fused by scheduler.